### PR TITLE
Frontend wip

### PR DESCRIPTION
**Pull request by phlpphcknr**
- build frontend page artist overview
- recommendation section still wip
- small adjustments/necessary fixes in backend

### CODE REVIEW

File: frontend/src/components/ReleaseDescription.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            return <h3>{release.releaseYear}</h3>
+        }
+    }
+
+    return (
+        <Release>
+            <img src={release.coverImageUrl} alt={'Shows the release cover'}/>
+            <section>
+                <h3>{release.fullTitle}</h3>
+                {renderReleaseYear()}
```
Comment (christophersiem) [inline frontend/src/components/ReleaseDescription.js:18]: this way you don't need the renderReleaseYear function

```suggestion
                {release.releaseYear !== 0 &&  <h3>release.releaseYear</h3>}
```

File: frontend/src/components/ReleaseDescription.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+}
+
+
+
+const Release = styled.section`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px;
+  margin: 0px 20px 0px 20px;
```
Comment (christophersiem) [inline frontend/src/components/ReleaseDescription.js:31]:
```suggestion
  margin: 0px 20px;
```

**Review by christophersiem — APPROVED** [review]
Nice 🙌🏽

File: frontend/src/components/ReleaseDescription.js (hunk 1 above)
Comment (phlpphcknr) [reply frontend/src/components/ReleaseDescription.js:18]: thx. changed ...

File: frontend/src/components/ReleaseDescription.js (hunk 2 above)
Comment (phlpphcknr) [reply frontend/src/components/ReleaseDescription.js:31]: I changed it and it seems fine ... for my understanding: just using two values for margin, the first one specifies top & bottom and the second one specifies right & left ?!

File: frontend/src/components/GlobalStyle.js [hunk 3]
```diff
@@ -1,18 +1,28 @@
 import { createGlobalStyle } from 'styled-components'
 
 export default createGlobalStyle`
-  * {
-    box-sizing: border-box;
-  }
-
-  html, body {
+  html, body, #root {
```
[the remark this answers is not in the document]
Comment (phlpphcknr) [reply frontend/src/components/GlobalStyle.js:4]: thx. I fixed this

File: frontend/src/components/ArtistRecommendation.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components/macro'
+
+export default function ArtistRecommendation ({onRecommend, artistName}){
```
[the remark this answers is not in the document]
Comment (phlpphcknr) [reply frontend/src/components/ArtistRecommendation.js:3]: ok. I will follow this in the future.

File: backend/src/test/java/com/hackner/musicompass/helper/TestData.java [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+
+public class TestData {
+
+    public Artist getArtist(String artistName, Date saveDate){
+        //Date saveDate = Date.from(timeUtils.now());
```
[the remark this answers is not in the document]
Comment (phlpphcknr) [reply backend/src/test/java/com/hackner/musicompass/helper/TestData.java:17]: thx. changed

File: frontend/src/components/ReleaseDescription.js (hunk 2 above)
Comment (christophersiem) [reply frontend/src/components/ReleaseDescription.js:31]: correct 👍🏽

**Pull request merged**